relay-spans/src/span.rs

```diff
@@ -84,6 +84,7 @@ pub fn otel_to_sentry_span(otel_span: OtelSpan) -> EventSpan {
     let trace_id = hex::encode(trace_id);
     let parent_span_id = hex::encode(parent_span_id);
 
+    // TODO: This is wrong, a segment could still have a parent in the trace.
```

Comment:
Will fix this in a different PR. It will not skew the measurements by much because we barely ingest any standalone spans from SDKs yet.